package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "marked-smartypants",
-  "version": "1.0.5",
+  "version": "1.1.0",
```

Comment:
You don't need to update this version. It will be automatically updated when we release a new version.